spec/classes/software_vcsscm_git_spec.rb

```diff
@@ -15,6 +15,65 @@
           it { is_expected.to contain_package('git') }
         end
       end
+
+      context 'with booleans for gitconfig and gitignore'
+        let :params do
+          {
+            :gitconfig => true,
+            :gitignore => true,
+          }
+        end
+        if facts[:operatingsystem] == 'Darwin'
+          it { is_expected.to compile.and_raise_error(/is not supported on Darwin./) }
+        else
+          it { is_expected.to compile.with_all_deps }
+          it { is_expected.to contain_package('git') }
+          it { is_expected.to contain_file('/etc/gitconfig') }
```

Comment:
How about testing that the source of these files changes as expected?
http://rspec-puppet.com/documentation/classes/#test-resource-parameters

Reply:
```ruby
it { is_expected.to contain_file('/etc/gitconfig').with_source('puppet:///modules/software/vcsscm/git/system-gitconfig') }
```

Reply:
:+1: